pkg/source/adapter/adapter.go

```diff
@@ -95,10 +91,13 @@ func (a *Adapter) start(stopCh <-chan struct{}) error {
 	if err != nil {
 		return fmt.Errorf("failed to create the config: %w", err)
 	}
-	config.Consumer.Offsets.AutoCommit.Enable = false
+
+	// Autocommit offset marked as processed. This is usually faster
+	// than committing after each message has been processed
+	config.Consumer.Offsets.AutoCommit.Enable = true
```

Comment:
@pierDipi I think you originally were setting this to `false`

https://github.com/slinkydeveloper/eventing-contrib/commit/a667d57d8b5b04ac5153e22db45a36497b4abbc6

Reply:
I think it's safe to enable auto-commit since sarama keeps track of the latest marked offset and it commits that at specified intervals, so we should keep at-least-once delivery semantics anyway.
Although this increases the probability of processing messages multiple times, so it depends on what we want to optimize.

Reply:
How about... making this _configurable_ ?  As the setting comes with Pros/Cons :man_shrugging:

Reply:
yes this should be configurable, not only AutoCommit, but everything else (?) relevant to KafkaSource. However the current implementation does not commit offsets at all so I just pick one policy.

Reply:
Do we really need to do this change in this PR?

Reply:
yeah, I was wondering the same.

@lionelvillard for the configruation enhancements: can you file an issue?

Reply:
It's a bug fix we don't commit the offset at all, so it's better than nothing.

Reply:
@matzew done: https://github.com/knative-sandbox/eventing-kafka/issues/108

Reply:
done in a separate PR.